cogeo_mosaic_tiler/handlers/app.py

```diff
@@ -627,7 +628,7 @@ def _img(
     return (
         "OK",
         f"image/{ext}",
-        array_to_image(rtile, mask, img_format=driver, color_map=color_map, **options),
+        render(rtile, mask, img_format=driver, color_map=color_map, **options),
```

Comment:
`color_map` needs to be changed to `colormap`

Reply:
@vincentsarago you should change this if you didn't already

Reply:
yep https://github.com/developmentseed/cogeo-mosaic-tiler/commit/8aacee9ba89985b01adeca05b48f5c459972dce3